tests/fields/test_decimal128_field.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from bson.decimal128 import Decimal128
+
+from mongoengine import *
+from tests.utils import MongoDBTestCase
+
+
+class TestDecimal128Field(MongoDBTestCase):
```

Comment:
I've also seen some code for dealing with int/float so this should be checked as well in another test

Reply:
82d45f14bc193f7f230a6cc333f29a07a4173a48